src/auth/providers.js

```diff
@@ -44,6 +47,9 @@ function userPassProvider(auth) {
   return {
     /**
      * Login to a stitch application using username and password authentication
+     *
+     * @param {String} username the username to use for authentication
+     * @param {String} password the password to use for authentication
```

Comment:
Similar to above comment: `@returns {Promise} a promise that resolves when authentication succeeds.